client/setup.py

```diff
@@ -53,7 +53,7 @@ def run_tests(self):
       license='LGPLv3',
       long_description=open('../readme.md', 'r').read(),
       name='dpcs',
-      packages=[],
+      packages=['dpcs_client'],
```

Comment:
I couldn't find a better option, thus we'll stick with a package for utilities/modules.